src/app/personal/personal.component.html

```diff
@@ -1,9 +1,256 @@
 <div class="warranty-wiz--content">
-		<h1 class="warranty-wiz--header">Personal Information</h1>
-		<h2 class="warranty-wiz--subheader">First, tell us about yourself.</h2>
-		<p class="warranty-wiz--intro">We need this information so we know how to get back with you. We keep all of this strictly confidential.</p>
-		<p class="warranty-wiz--intro">*Please note that photos will be required of the items and issues you are experiencing.  If these cannot be attached to this claim, we ask that you call or chat with Customer Care to place your claim. Customer Care is available at 1-800-748-5400 or through the Live Chat (M-F, 7am-6pm MST)</p>
+		<h1 class="warranty-wiz--header">{{'HEADING.PERSONAL_INFORMATION' | translate}}</h1>
+		<h2 class="warranty-wiz--subheader">{{'LABEL.TELL_ABOUT_YOURSELF' | translate}}</h2>
+		<p class="warranty-wiz--intro">{{'CONTENT.CONFIDENTIAL' | translate}}</p>
+		<p class="warranty-wiz--intro">{{'CONTENT.PHOTO_INFO' | translate}}</p>
 
+    <form novalidate (ngSubmit)="onSubmit(personal)" [formGroup]="personal" class="animate-floating-labels">
+      <!-- TODO - more elegant solution to massive if -->
+      <div class="errors" *ngIf="personal.touched &&
```

Comment:
This is my fault, but I hate it. Look into registration there is a util function that loops through the form and pulls error count for those touched.